### PR TITLE
1188: cleanup, only return necessary fields from case search

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: web-api/cases.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+
+1513-19 6b508fb8-46c7-41b1-82f2-6ab0cea5c0e4
```
Comment (codyseibert) [inline web-api/cases.txt]: could we git ignore / `git -rm` this file please

File: web-api/create-cases.sh [hunk 2]
```diff
@@ -109,91 +109,11 @@ EOF
...
-  curl "https://efcms-${ENV}.${EFCMS_DOMAIN}/api/run-batch-process" \
-    -H 'Accept: application/json, text/plain, */*' \
-    -H "Authorization: Bearer ${petitionsclerkToken}" \
-    -H 'Content-Type: application/json;charset=UTF-8' \
-    --data-binary '{}' \
-    --compressed
-
-  curl "https://efcms-${ENV}.${EFCMS_DOMAIN}/cases/${caseId}/set-to-ready-for-trial" \
-    -X PUT \
+  curl "https://efcms-${ENV}.${EFCMS_DOMAIN}/cases/${caseId}/block" \
```
Comment (codyseibert) [inline web-api/create-cases.sh]: I would revert this file; setting stuff to blocked was mainly for us testing this story

File: web-api/ux_testing_data.csv [hunk 3]
```diff
@@ -1,70 +1,70 @@
...
+Petitioner;Deficiency;Sophie Taliaferro;;Regular;Jackson, Mississippi;respondent1;practitioner1
+Petitioner;Deficiency;Izetta Laakso;;Regular;Jackson, Mississippi;respondent1;practitioner2
+Petitioner;Deficiency;Cleveland Gamez;;Regular;Jackson, Mississippi;respondent1;
+Petitioner;Deficiency;Shon Ewen;;Regular;Jackson, Mississippi;respondent1;
+Petitioner;Deficiency;Marnie Roof;;Regular;Jackson, Mississippi;respondent2;
+Petitioner;Deficiency;Chris Easterday;;Regular;Jackson, Mississippi;respondent2;
+Petitioner;Deficiency;Malena Burrowes;;Regular;Jackson, Mississippi;respondent2;
+Petitioner;Deficiency;Gerard Maycock;;Regular;Jackson, Mississippi;respondent2;
+Petitioner;Deficiency;Pinkie Dorsey;;Regular;Jackson, Mississippi;respondent2;
+Petitioner;Deficiency;Buford Brunell;;Regular;Jackson, Mississippi;respondent2;
```
Comment (codyseibert) [inline web-api/ux_testing_data.csv]: I'd also revert this file

**Review by wbyoko — APPROVED** [review]

File: shared/src/business/entities/cases/Case.js [hunk 4]
```diff
@@ -357,7 +357,6 @@ joiValidationDecorator(
     receivedAt: joi
       .date()
       .iso()
-      .max('now')
```
Comment (rachaelparris) [inline shared/src/business/entities/cases/Case.js:360]: This is causing problems when tests are being run late in the day PST. I think we might have something weird going on with validation of dates that are on the following day when converted to UTC. `createdAt` doesn't have this restriction, so I've removed it here, but we may need to look into this further.

**Pull request merged**